docs/source/conf.py

```diff
@@ -1,10 +1,11 @@
 """Sphinx Documentation configuration."""
+
 import os
 import sys
+import tomllib
 from typing import List
 
 import sphinx_readable_theme
-import toml
```

Comment:
no idea where toml was coming from, didn't see it in the dependencies directly which it ought to have been, anyway, tomllib is built in now in >= 3.11 thankfully.